adc_manager/src/adc_manager.py

```diff
@@ -0,0 +1,35 @@
+from MCP3008 import MCP3008
+from datetime import datetime
+import json  
+import time
+
+
+adc = MCP3008()
+filename = '../test_logs/log_test.json'              # Name of log fie
```

Comment:
only /src/* will get copied into /usr/bin/, if you need a template .json, should put it in the /src/ directory